src/lib/components/chat/SettingsModal.svelte

```diff
@@ -330,11 +335,10 @@
 
 				if (!data.success) {
 					toast.error(data.error);
-					return;
+				} else {
+					toast.success(`Model ${modelName} was successfully downloaded`);
```

Comment:
suggestion: the idea is not to report toast notification from the queue processor, instead return the final state and handle them from the job completion callback. So UI logic and business logic don't get mixed together